multimodal/tests/unittests/others/test_deployment_onnx.py

```diff
@@ -185,12 +185,7 @@ def test_onnx_optimize_for_inference(dataset_name, model_names, text_backbone, i
         predictor_opt = MultiModalPredictor.load(path=model_path)
         predictor_opt.optimize_for_inference(providers=providers)
 
-        # Check existence of the exported onnx model file and tensorrt cache files
-        onnx_path = os.path.join(predictor_opt.path, "model.onnx")
-        assert os.path.exists(onnx_path), f"onnx model file not found at {onnx_path}"
-        if providers == None or providers == ["TensorrtExecutionProvider"]:
-            trt_cache_dir = os.path.join(predictor_opt.path, "model_trt")
-            assert len(os.listdir(trt_cache_dir)) >= 2, f"tensorrt cache model files are not found in {trt_cache_dir}"
```

Comment:
We probably want to test the logic for `export_onnx` when path is provided as well.

Reply:
test logic added.